services/API-service/src/api/notification/email/mjml/body-event.ts

```diff
@@ -61,15 +64,24 @@ const getMjmlBodyEvent = ({
 
   const contentContent = [];
 
-  contentContent.push(
-    firstTriggerLeadTimeString
-      ? `<strong>${disasterTypeLabel}:</strong> Expected to start on ${firstLeadTimeString}, ${firstLeadTimeFromNow}.`
-      : `<strong>${disasterIssuedLabel}:</strong> Expected on ${firstLeadTimeString}, ${firstLeadTimeFromNow}.`,
-  );
-
-  if (firstTriggerLeadTimeString) {
+  if (disasterType === DisasterType.Floods) {
```

Comment:
@gulfaraz I agree with the preference. What I did in this commit is simply align the (incorrect) logic with the correct logic in the chat section of the portal, which unfortunately also uses this exception on floods.

That said, I'm happy to look at what we can do, but your above suggestion is missing one thing. Namely, the bottom 2 cases do not only differ on the phrase 'to start' but also on 'disasterIssuedLabel' vs ' disasterTypeLabel'.  

I will take another look to see what's possible.

Reply:
You're right, I missed the _disasterIssuedLabel_ vs _disasterTypeLabel_ difference.

It would be great if the check is not specific to a hazard or country.

Reply:
@gulfaraz see new commit. I made use of the fact that disasterIssuedLabel is based on eapAlertLabel, which is only filled for floods. If not filled I default to disasterTypeLabel, which allows for the combining of above scenarios. The previous fallback option was never used in practice anyway. I also added some code comments, as I think that doesn't hurt for this relatively complex logic. Ok?